packages/php-db-import-export/src/Backend/Snowflake/Export/AbsExportAdapter.php

```diff
@@ -39,6 +39,10 @@ public function runCopyCommand(
         Storage\DestinationInterface $destination,
         ExportOptionsInterface $exportOptions,
     ): array {
+        $timestampFormat = 'YYYY-MM-DD HH24:MI:SS';
+        if (in_array('snowflake-export-fractional-seconds', $exportOptions->features(), true)) {
```

Comment:
Můžeme mít `'snowflake-export-fractional-seconds'` jako nějakou konstantu někde? Nebo ještě lépe nějaký ENUM.

Opakuje se 3x v kódu a jednou v testech.

Aspoň půjde lehce vidět, kde všude je to použité.

Reply:
može byt nebude tam string v těch adaptérech jinde to využití už nemá pak. Do testu sem to nedával ten by měl spadnůt když sa to změní